src/BridgeHelper/CatBoostLibraryBridgeHelper.h

```diff
@@ -17,9 +18,15 @@ class CatBoostLibraryBridgeHelper : public LibraryBridgeHelper
     static constexpr inline auto MAIN_HANDLER = "/catboost_request";
 
     CatBoostLibraryBridgeHelper(ContextPtr context_, std::string_view library_path_, std::string_view model_path_);
+    CatBoostLibraryBridgeHelper(ContextPtr context_, std::string_view model_path_);
+    explicit CatBoostLibraryBridgeHelper(ContextPtr context_);
 
-    size_t getTreeCount();
+    ExternalModelInfos listModels();
+
+    void removeModel();
+    void removeAllModels();
```

Comment:
this seems a little inconsistent that model_path can be an argument in bridge helpers constructor and at the same not be, e.g. some methods need to ensure that bridge helper was created with a certain constructor. May be rework it somehow to avoid such complication?